events/README.md

```diff
@@ -106,6 +108,7 @@ A Curb Event is represented as a JSON object, whose fields are as follows:
 | `curb_zone_id` | [UUID][uuid] | Conditionally Required | Unique ID of the Curb Zone where the event occurred. Required for events that occurred at a known Curb Zone for ALL _event_types_. |
 | `curb_area_ids` | Array of [UUID][uuid] | Conditionally Required | Unique IDs of the Curb Area where the event occurred. Since Curb Areas can overlap, an event may happen in more than one. Required for events that occurred in a known Curb Area, if known and used, for these event_types: _enter_area, exit_area, park_start, park_end_ |
 | `curb_space_id` | [UUID][uuid] | Conditionally Required | Unique ID of the Curb Space where the event occurred. Required for events that occurred at a known Curb Space, if known and used, for these event_types: _park_start, park_end, enter_area, exit_area_ |
+| `curb_object_id` | [UUID][uuid] | Conditionally Required | Unique ID of the Curb Object where the event occurred. Required for events that occurred at a known Curb Object, if known and used, for these event_types: _park_start, park_end, enter_area, exit_area_ |
```

Comment:
Should this be for multiple IDs, so `curb_object_id`? One event could use a meter and an EV charger and a locker, for example.

Reply:
We could list multiple IDs here, although maybe it should only be what the true source of the event is (if you plug into the charger, book the locker, or pay at the meter)? I think leaving it as one option could be cleaner.

Reply:
Agreed that this can stay as one single ID per event. If multiple objects need to record simultaneous events at a zone, then they each get their own event. So this is fine as is.